### PR TITLE
0.9 release

### PR DESCRIPTION
**Pull request by keis**
draft news item for 0.9

ping @mathstuf :)

### CODE REVIEW

File: news/41.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+2016.xx.xx: uzbl 0.9
+
+With a lot of big refactoring and smaller patches piling up over the past years
+this release is, needless to say, way overdue.
+
+So what's with the version number? Why not 1.0?  We wanted to make a 1.0 the
+version with fully supported WebKit2, but this is not that. In fact webkit2
```
Comment (mathstuf) [inline news/41.txt]: webkit2 → WebKit2

File: news/41.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* And various assorted bug fixes
+
+The new <a href="https://github.com/uzbl/uzbl/releasen/tag/0.9.0">release</a>
+is available on github. Also make sure to read the <a
+href="https://github.com/uzbl/uzbl/master/UPGRADING.md>upgrading
+instructions</a> if upgrading from earlier versions.
+
+Looking ahead webkit2 is the next big challenge and the plan is to start
+building uzbl 1.0 in the next branch that will only support webkit2 but still
+with the same api towards scripts etc as this release.
```
Comment (mathstuf) [inline news/41.txt]: api → API; webkit2 → WebKit2, "etc" → "etc.".

**Comment by mathstuf** [comment]
Other than that, LGTM.

**Comment by keis** [comment]
Ok. thanks :)

and a note for self. remember to put the proper links in when the github release is tagged.

**Comment by mathstuf** [comment]
Hmm, should also put release process docs in uzbl itself.

**Comment by The-Compiler** [comment]
https://github.com/uzbl/uzbl/master/UPGRADING.md is a 404, probably should be https://github.com/uzbl/uzbl/blob/master/UPGRADING.md :wink:

**Comment by keis** [comment]
oops. yes. fixing

**Pull request merged**